java/code/src/com/redhat/rhn/domain/errata/ErrataFactory.java

```diff
@@ -494,17 +419,9 @@ private static void copyDetails(Errata copy, Errata original, boolean clone) {
         Iterator bugsItr = IteratorUtils.getIterator(original.getBugs());
         while (bugsItr.hasNext()) {
             Bug bugIn = (Bug) bugsItr.next();
-            Bug cloneB;
-            if (copy.isPublished()) { //we want published bugs
-                cloneB = ErrataManager.createNewPublishedBug(bugIn.getId(),
-                        bugIn.getSummary(),
-                        bugIn.getUrl());
-            }
-            else { //we want unpublished bugs
-                cloneB = ErrataManager.createNewUnpublishedBug(bugIn.getId(),
+            Bug cloneB = ErrataManager.createNewPublishedBug(bugIn.getId(),
```

Comment:
I'm confused with this one. The goal is to remove unpublished errata, but here we keep the code of the unpublished bugs and remove the code block for the published ones.

Reply:
I think the change is correct.

Original code said: `if (copy.isPublished())` ... `createNewPublishedBug` ... `else` ... `createNewUnpublishedBug`

New code says: `createNewPublishedBug`

(assumption is it can't be unpublished, so it's always published)